docs: updated docs for internationalization (#6736)
vuestorefront/vue-storefront: packages/core/docs/getting-started/internationalization.md

        iso: 'de',
      },
    ],
    defaultLocale: 'en',
  },
};
```

## Configuring modules separately

You can provide your own i18n configuration for a specific module by setting `useNuxtI18nConfig` to `false`.

```js
// nuxt.config.js

export default {
  [
    '@vue-storefront/{INTEGRATION}/nuxt',
    {
      i18n: {
        useNuxtI18nConfig: false,
        locales: [
          {
            code: 'en',
            label: 'English',
            file: 'en.js',
            iso: 'en',
          },
          {
            code: 'de',
            label: 'German',
            file: 'de.js',
            iso: 'de',
          },
        ],
        defaultLocale: 'en'
      }
    }
  ];
}
```

## CDN and cookies

To make CDN to work correctly, it cannot return `Set-Cookie` header in response.

To achieve that, we've made our own mechanism for handling cookies responsible for
storing `locale`, `currency` and `country`.

This mechanism also handles language detection and redirects for proper locale versions.
This is why `@nuxtjs/i18n` language detection is disabled by default.

```js
// nuxt.config.js
export default {
  i18n: {
    detectBrowserLanguage: false,
  },
};
```

> If you don't want to redirect users, you can disable this mechanism, as described in the [section below](#disabling-the-auto-redirect-mechanism).

Author: That is correct. Link was added just in case that the order of section will change.  It's not needed right now.

```js
// nuxt.config.js

export default {
  i18n: {
    vueI18n: {
      numberFormats: {
        en: {
          currency: {
            style: 'currency',
            currency: 'USD',
            currencyDisplay: 'symbol',
          },
        },
        // ...other locales
      },
    },
  },
};
```

**Please note that only one currency can be set for each locale.**

If this is a limitation for you, or if you don't want to have currencies tied to locales, you can disable this mechanism and provide your own.

To disable it, set `autoChangeCookie.currency` to `false` as described in the [section below](#configuring-the-auto-cookie-change).

## Configuring the Auto Cookie Change

You can configure how `@vue-storefront/nuxt` handles cookie cookie changes when the locale changes with the `autoChangeCookie` object.

The `autoChangeCookie` object holds three properties that are direct linked to `currency`, `locale` and `country`. These properties control how the module will handle cookie changes.

> If set to `false`, the module won't change any cookie based on configurations or browser locale, so the `{INTEGRATION}` will handle that.

Reviewer: ```suggestion
If set to `false`, the module won't change any cookies based on configurations or browser locale, and the integration will have to handle it.
```

```js
// nuxt.config.js

export default {
  buildModules: [
    [
      '@vue-storefront/{INTEGRATION}/nuxt',
      {
        i18n: {
          useNuxtI18nConfig: true,
        },
      },
    ],
  ],
  i18n: {
    locales: [
      {
        code: 'en',
        label: 'English',
        file: 'en.js',
        iso: 'en',
      },
      {
        code: 'de',
        label: 'German',
        file: 'de.js',
        iso: 'de',
      },
    ],
    defaultLocale: 'en',
    autoChangeCookie: {
      currency: false,
      locale: false,
      country: false,
    },
  },
};
```

## Configuring cookie attributes

> You can overwrite the default `locale`, `currency` and `country` cookie attributes set by `@vue-storefront/nuxt` or add custom attributes.
> 
> Just create a `cookieOptions` object for the `i18n` configuration object in the `nuxt.config.js` file and specify the attributes you'd like the cookies to have.

Reviewer: ```suggestion
You can overwrite the default `locale`, `currency`, and `country` cookie attributes set by `@vue-storefront/nuxt` or add custom attributes.

To do so, add the `cookieOptions` object to the `i18n` configuration object in the `nuxt.config.js` file and specify the attributes you'd like the cookies to have.
```
